base/sysimg.jl

```diff
@@ -115,9 +115,9 @@ include("intfuncs.jl")
 include("nullable.jl")
 
 # I/O
+include("show.jl")
 include("task.jl")
 include("lock.jl")
-include("show.jl")
```

Comment:
This seems like a gratuitous move of when `show.jl` is loaded in the system image.